<launch>
    <node name="point_cloud_edit" pkg="table_object_detector" type="pc_merge" output="screen" respawn="true">        
    </node>
</launch>

